bundles/org.openhab.binding.bluetooth.dbusbluez/src/main/java/org/openhab/binding/bluetooth/dbusbluez/handler/DBusBlueZBridgeHandler.java

```diff
@@ -0,0 +1,343 @@
+        logger.debug(
+                " | |  | | |_) |_   _ ___  | |_) | |_   _  ___   / /  | |_) | |_   _  ___| |_ ___   ___ | |_| |__   ");
+        logger.debug(
+                " | |  | |  _ <| | | / __| |  _ <| | | | |/ _ \\ / /   |  _ <| | | | |/ _ \\ __/ _ \\ / _ \\| __| '_ \\  ");
+        logger.debug(
+                " | |__| | |_) | |_| \\__ \\ | |_) | | |_| |  __// /__  | |_) | | |_| |  __/ || (_) | (_) | |_| | | | ");
+        logger.debug(
+                " |_____/|____/ \\__,_|___/ |____/|_|\\__,_|\\___/_____| |____/|_|\\__,_|\\___|\\__\\___/ \\___/ \\__|_| |_| ");
+        logger.debug("");
+
+        logger.debug("Initializing DBusBlueZBridgeHandler...");
+
+        initializeInternal();
+
+    }
+
+    /**
+     * This function initializes the library
+     *
+     * @return True if succeeded initializing the library, false otherwise
+     * @throws InterruptedException
+     */
+    private boolean initializeDeviceManager() {
+        logger.debug("Initializing Device Manager");
+
+        try {
+            initializeDeviceManagerInternal();
+        } catch (DBusException e1) {
+            logger.error("failed create instance caused by D-BUS.", e1);
+            return false;
+        }
+
+        try {
+            Thread.sleep(100);
+        } catch (InterruptedException e1) {
+            Thread.currentThread().interrupt();
+        }
+
+        // This call will return for sure the DM, otherwize createInstance
+        // would have failed previously
+        this.deviceManager = DeviceManager.getInstance();
+
+        if (this.deviceManager != null) {
+            logger.debug("Device Manager correctly instanciated");
+
+            // a handler must be instanciated to get all notifications
+            // from DBUS (new device, RSSI update, characteristic notification...)
+            try {
+                this.deviceManager.registerPropertyHandler(this.propertiesChangedHandler);
+            } catch (DBusException e) {
+                // Shoudl not happen..
+                logger.error("Error registering properties changed handler", e);
+                return false;
+            }
+        } else {
+            // should normally not happen..
+            logger.debug("Device Manager could not be instanciated but no error.");
+        }
+
+        return (this.deviceManager != null);
+    }
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+
+    private void initializeDeviceManagerInternal() throws DBusException {
+        try {
+            // if this is the first call to the library, this call
+            // should throw an exception (that we are catching)
+            this.deviceManager = DeviceManager.getInstance();
+
+            // Experimental - seems reuse does not work
+            this.deviceManager.closeConnection();
+            DeviceManager.createInstance(false);
```

Comment:
If the DeviceManager is shared between handler instances I don't think it is very safe to close them when a handler initializes.